modules/hydrodyn/src/HydroDyn_Input.f90

```diff
@@ -3011,6 +3037,11 @@ SUBROUTINE HydroDynInput_ProcessInitData( InitInp, ErrStat, ErrMsg )
          RETURN
       END IF
 
+      if ( (.not. ( EqualRealNos(InitInp%DT, InitInp%WAMIT%Conv_Rdtn%RdtnDT) ) ) .and. ( (InitInp%WAMIT%ExctnMod > 1) .or. (InitInp%WAMIT%RdtnMod > 0) ) ) then
+         call SetErrStat( ErrID_Fatal,'RdtnDT must be equal to the glue-code DT if PotMod = 1 and using RdtnMod > 0 or ExctnMod > 1.',ErrStat,ErrMsg,RoutineName)
+         return
+      end if
+      
```

Comment:
@ghaymanNREL, @jjonkman : This new check prevents the ITIBarge regression test from running. Is this the correct logic? Or were the old input files set up improperly?

`RdtnDT must be equal to the glue-code DT if PotMod = 1 and using RdtnMod > 0 or ExctnMod > 1`

Reply:
Has this been resolved?

Reply:
@ghaymanNREL sent me some email chains about this issue in February. After reading them, I inferred that the previous implementation of the `RdtnDT` time step was incorrect if it wasn't the same as the glue code, so they added this check. I changed the input file that was causing issues in the regression tests for PR #373 (see https://github.com/bjonkman/r-test/commit/c2fe16a9ccf2000034c615bbe93d33599fb4f358). Greg probably modified it in his r-test branch, too.

Actually, it looks like @jjonkman wanted the comment for `RdtnDT` to be updated to read
```
 Time step for wave radiation kernel calculations (sec) [only used when PotMod=1 and RdtnMod=1; DT<=RdtnDT<=0.1 recommended; determines RdtnOmegaMax=Pi/RdtnDT in the cosine transform]
```
Guess we should update that in the HD input files, too.

Reply:
I think @rafmudaf created a new r-test branch for this PR from a commit on the r-test/bjonkman-linear branch.

Reply:
I noticed that @ghaymanNREL already had a branch with these changes, too (last October/November), so I made a pull request for @rafmudaf to merge that into his new branch.